Voycar.Api.Web/Features/Reservation/Endpoints/Get/Personal/Endpoint.cs

```diff
@@ -0,0 +1,65 @@
+namespace Voycar.Api.Web.Features.Reservation.Endpoints.Get.Personal;
+
+using Entities;
+using Repository;
+using Users.Repository;
+
+
+public class Endpoint : Endpoint<Request, Results<Ok<Response>, BadRequest<string>>>
+{
+    private readonly IUsers _userRepository;
+    private readonly IReservations _reservationsRepository;
+
+
+    public Endpoint(IUsers userRepository, IReservations reservationsRepository)
+    {
+        this._userRepository = userRepository;
+        this._reservationsRepository = reservationsRepository;
+    }
+
+
+    public override void Configure()
+    {
+        this.Get("reservation/personal");
+        this.Roles("member");
+        this.Summary(summary =>
+        {
+            summary.Summary = "Get reservations of logged in member";
+            summary.Description = "Get the reservations of the member who is logged in according to the request cookie";
```

Comment:
```suggestion
            summary.Summary = $"Get {nameof(Reservation)}s of logged in member";
            summary.Description = $"Get the {nameof(Reservation)}s of the member who is logged in according to the request cookie";
```

In einem Versuch das etwas weniger zu hardcoden hatte ich das für `/car/available` so gemacht.